utils/container-runtime/tests/tests/test_setup.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use container_runtime::ContainerRuntime;
+
+use super::test_container::TEST_IMAGE;
+
+// Not really a test - used by CI to separate pulling of test images
+// into its own phase
+#[cfg_attr(feature = "skip_docker_tests", ignore)]
+#[test_log::test(tokio::test)]
+async fn test_setup_pull_images() {
```

Comment:
This *might* help to stop my container test from failing - it appears to be some race condition in signal handling in `podman` itself that allows the controlling attached process of `podman` to exit while container is still running. Last time I saw image pull happening in parallel (by another test) so I want to make sure that we pull all images before test execution starts, as a separate CI step.